.github/workflows/requirements-lint.txt

```diff
@@ -0,0 +1,67 @@
+#
+# This file is autogenerated by pip-compile with Python 3.11
+# by the following command:
+#
+#    pip-compile --generate-hashes requirements-lint.in
```

Comment:
Can we make these requirement files top-level files so that they can be used locally when running linters? e.g. via make

Reply:
moved